gradle-baseline-java-config/resources/checkstyle/checkstyle.xml

```diff
@@ -269,6 +280,42 @@
             <property name="format" value="^import org\.apache\.commons\.math\."/>
             <property name="message" value="math is deprecated, use math3 instead"/>
         </module>
+        <module name="RegexpSinglelineJava">
+            <property name="format" value="\bPreconditions\.check.*\{\}"/>
```

Comment:
This is awesome. Though "Use printf-style formatting in Preconditions, not-{} style formatting" might be better.